src/platform/tests/TestPlatformMgr.cpp

```diff
@@ -49,24 +47,41 @@ using namespace chip::DeviceLayer;
 //      Unit tests
 // =================================
 
-static void TestPlatformMgr_InitShutdown(nlTestSuite * inSuite, void * inContext)
+class TestPlatformMgr : public ::testing::Test
+{
+public:
+    static void SetUpTestSuite()
+    {
+        CHIP_ERROR error = chip::Platform::MemoryInit();
+        EXPECT_EQ(error, CHIP_NO_ERROR);
+
+        // Setup a fake commissionable data provider since required by internals of several
```

Comment:
```suggestion
        // Set up a fake commissionable data provider since required by internals of several
```
I know you just copies this....